### ISSUE TITLE
Check Bridge Foundry Phone Number

### ISSUE THREAD

**Issue by carriebridgefoundry** [bridgefoundry/operations#138]
This phone number was at the bottom of our marketing email. It has no voicemail or identifier. We could either add a voicemail/identifier, or do something else completely with the number.

224-801-2358

**Comment by mxie** [comment]
Good catch! Let’s update the number to (909) 703-5303. This is our new Google Voice number since we lost our old one.

**Comment by carriebridgefoundry** [comment]
Tested the new number! It needs a voicemail message that is relevant. Left a message, waiting to see who calls back.
